bot/potentialScammer/getMembersUsername.module.js

```diff
@@ -0,0 +1,26 @@
+const { fetchMatchConditionMembers } = require(
+  './fetchMatchConditionMembers.module')
+
+const getMembersKeyInformation = (members) => {
+  let membersKeyInformation = new Map()
+  members.each(member => {
+    membersKeyInformation.set(
+      member.user.id, {
+        username: member.user.username,
+        discriminator: member.user.discriminator
+      })
+  })
+  return membersKeyInformation
+}
+
+const getMembersUsername = async (guild, client) => {
+  const members = await fetchMatchConditionMembers(guild, client)
+  const membersKeyInformation = getMembersKeyInformation(members)
+  let membersUsername = []
+  for (let member of membersKeyInformation.values()) {
```

Comment:
Use `map` function instead of `for .. of`

Reply:
this is membersKeyInformation data structure:
![image](https://user-images.githubusercontent.com/53669064/174048260-c4e5087b-0ece-4b1d-8a1a-d5afcefbbfb8.png)
do you mean change the data structure?
membersKeyInformation.valuse()  can't suppost `map` function,because it's a Map Iteractor object

Reply:
Ok, got it.